packages/frontend/src/components/Sidebar/RequirementSection.tsx

```diff
@@ -8,6 +8,7 @@ import {
   Requirement,
   ScheduleCourse,
 } from "@graduate/common";
+import { fetchCourse } from "../../api";
```

Comment:
there are an awful lot of import and formatting changes in here– in the future, try to avoid changing files that you aren't touching functionally